net-analyzer/speedtest_cli/speedtest_cli-2.0.2.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="A CLI for testing internet bandwidth using speedtest.net"
+DESCRIPTION="speedtest-cli is a command-line interface for testing internet \
+bandwidth using speedtest.net. Note that latency reported by this tool should \
+not be relied on as a value indicative of ICMP style latency; rather, it is a \
+relative value used for determining the lowest latency server for performing \
+the actual speed test against.
+Also note that its results may also be inconsistent with speedtest.net due to \
+the website's migration to using pure socket tests instead of HTTP based ones, \
+the application being written in Python, \
+differing performance between Python versions, \
+and CPU and Memory capacity and speed."
+HOMEPAGE="https://github.com/sivel/speedtest-cli"
+COPYRIGHT="2012-2018 Matt Martz"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="5e22f2dcce1c9020f33faf559b22727483f022008a2912b85d573e578374b6a0"
+SOURCE_FILENAME="v$portVersion.tar.gz"
```

Comment:
Thanks @Crestwave for your PR!

Recommendation:
```diff
-SOURCE_FILENAME="v$portVersion.tar.gz"
+SOURCE_FILENAME="speedtest-cli-$portVersion.tar.gz"
```
([HaikuPorter-Guidelines#archive-name](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name))

Reply:
Could you please move the recipe from `net-analyzer/speedtest_cli` to `net-analyzer/speedtest-cli`, but keeping the name of the recipe unchanged? (`speedtest_cli-2.0.2.recipe`)
Thanks!

Reply:
Whoops; I did read that but I thought that SOURCE_FILENAME was for specifying it if it didn’t follow the standard for some reason. Is the dash for the directories a new custom? I looked through the tree first and they had underscores instead. Thanks for the corrections!

Reply:
You're welcome! That's right. We only need `SOURCE_FILENAME` if the original name does not contain a meaningful basename.
For some recipes, e.g. sg3_utils, we do have an underscore in both the directory and the name of the recipe, but that's because the name of that project really has an underscore.
That said, we do have quite some recipes in directories having an underscore in their names while they could be renamed to use a `-`. I guess we did not move them because doing so makes it harder to track the full git log for a port.
(For new recipes we like to use the best name.)

Reply:
No, I originally thought that you would use it to specify the downloaded filename in that situation; doesn’t make sense, I know. And thanks for the explanation!